kolibri/plugins/user/assets/src/views/SignInPage/index.vue

```diff
@@ -469,6 +469,9 @@
           };
           if (global.oidcProviderEnabled) {
             sessionPayload['next'] = this.nextParam;
+          } else if (this.$route.query.redirect && !this.nextParam) {
+            // Go to URL in 'redirect' query param, if arriving from AuthMessage
+            sessionPayload['next'] = this.$route.query.redirect;
```

Comment:
Hrm, any reason to use a separate `redirect` query param here, rather than just using `next` for both cases and removing the `global.oidcProviderEnabled` conditionality?

Reply:
I think the idea was that if both the `next` and `redirect` params were set, that `next` would take precedent. I'm not sure when or how `next` is set, so I wanted to use `redirect` to avoid a clash.